packages/hoppscotch-app/components/http/ResponseMeta.vue

```diff
@@ -141,6 +145,36 @@ const props = defineProps<{
   response: HoppRESTResponse
 }>()
 
+const responseSizeTippy = computed(() => {
+  if (
+    props.response.type === "loading" ||
+    props.response.type === "network_fail" ||
+    props.response.type === "script_fail" ||
+    props.response.type === "fail"
+  )
+    return
+  if (Number(props.response.meta.responseSize) <= 1000)
+    return {
+      onShow: () => {
+        return false
+      },
+    }
+  else return { theme: "tooltip", allowHTML: true }
+})
+
+const responseSizeHint = computed(() => {
+  if (
+    props.response.type === "loading" ||
+    props.response.type === "network_fail" ||
+    props.response.type === "script_fail" ||
+    props.response.type === "fail"
+  )
+    return
+  const size = Number(props.response.meta.responseSize)
```

Comment:
Redundant Number constructor. We are using TypeScript which guarantees `props.response.meta.responseSize` is a number.

```suggestion
  const size = props.response.meta.responseSize
```